### ISSUE TITLE
sius beluu

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#497]
Someone emailed asking about this. Is this a known expression?

**Comment by Gnalu** [comment]
Yes, it is. Like Beliasang ‘ menga sang! Ngeremlengui, menga cherremrum!
Those expressions

**Comment by Lukes-Imeyuns** [comment]
Is it SUIS or SIUS?

**Comment by johnbent** [comment]
What does it mean?

**Comment by Lukes-Imeyuns** [comment]
I'm thinking of what Mar said for the states.. But it's SIUS from MELIUS.

The SUIS Beluu made me think of a different word and wondered if the asker
misheard and misspelled...

**Comment by Gnalu** [comment]
Dialect: ‘I’ve heard both..’suusbeluu’ and ‘siusbeluu’

**Comment by johnbent** [comment]
Oh, it means, "slang expression by which to jokingly refer to people from a particular place (e.g. manga cherrerum to refer to people from Ngeremlengui)". Something like that?

**Comment by Gnalu** [comment]
Exactly!

**Comment by johnbent** [comment]
It would be nice to collect all of these actually. Can we do that?

Aimeliik - kmai
Airai - sang
Angaur
Hatohobei
Kayangel
Koror
Melekeok
Ngaraard - demo
Ngarchelong - kim, otkang
Ngardmau
Ngatpang
Ngchesar
Ngeremlengui - cherremrum
Ngiwal
Peleliu - tell
Sonsorol

Philipines
America - menga oats
Siabal

Then we could add 'SUIS BELUU' to our List tab. I do recognize this is triggering and offensive for at some people but I believe it is sill important (perhaps more important) to document.

**Comment by palaulanguage** [comment]
definition approved by PLC, 


Aimeliik - kmai
Airai - sang
Angaur- chechui 
Hatohobei- katsuo 
Kayangel- chiull /ngimer 
Koror- tiuach 
Melekeok-mud 
Ngaraard - irechiil 
Ngarchelong - kim, otkang
Ngardmau- senges 
Ngatpang- chemang 
Ngchesar- chum 
Ngeremlengui - cherremrum
Ngiwal- demok /Ebei 
Peleliu - telib
Sonsorol - uel 

Philipines- bakuchong 
America - menga oats
Siabal- menga takuang 



can also find these at https://tekinged.com/misc/pdf.php?file=odoim_ongraol 

PLC request to add a list for SIUS BELUU
